src/DecodeStream/DecodeStream.cpp

```diff
@@ -253,13 +252,21 @@ try { //Giant try block around all code to get error messages
         clog << "End of Sequence after " << frame << " frames, exiting" << endl;
       }
       return EXIT_SUCCESS;
+    case AUXILIARY_DATA:
+      if (d.length()<0) throw std::logic_error("Auxilliary data length is nonconformant.");
```

Comment:
Might be clearer to say 'Auxiliary data has length less than zero'?

Also, `s/Auxilliary/Auxiliary/`